docs/products/kafka/howto/prevent-full-disks.rst

```diff
@@ -0,0 +1,81 @@
+Prevent full disks
+===================
+
+To prevent **Aiven for Apache Kafka** from malfunctioning, the Aiven platform detects when the available disk space in **Aiven for Apache Kafka** services is running low.
+
+If any of the nodes in the service exceeds the critical threshold of disk usage, the access control list (ACL) that is used to authorize API requests by Apache Kafka clients is updated on all nodes to prevent operations that would increase disk usage. Such operations include:
+
+-  The ``Write`` and ``IdempotentWrite`` operations that clients use to produce new messages
+
+-  The ``CreateTopics`` operation that creates one or more topics, each carrying some overhead on disk
+
+When the ACL blocks write operations, clients see an error similar to the following example from the ``python-kafka`` client::
+
+   TopicAuthorizationFailedError: [Error 29] TopicAuthorizationFailedError: mytopic
+
+
+In ensure that enough disk space is available you can take one of the following actions:
+
+Upgrade to a larger service plan
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Log in to the `Aiven web console <https://console.aiven.io/>`_ and select your service.
+
+#. On the *Overview* page, scroll down to *Service plan* and click **Upgrade plan** .
+
+#. Select your new service plan and click **Upgrade** .
+
+
+This launches new nodes with more disk space. As data is moved from the old nodes to the new ones, disk usage returns to an acceptable level and the ACL is updated to allow write operations again.
```

Comment:
Should it be "**Once** data is moved from the old nodes..." the process needs to be completed before allowing writes again